LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/property-list.js

```diff
@@ -564,6 +564,10 @@ const gCSSProperties1 = {
     // https://drafts.csswg.org/css-fonts-4/#propdef-font-stretch
     types: [ 'percentage' ]
   },
+  'font-width': {
+    // https://drafts.csswg.org/css-fonts-4/#propdef-font-width
+    types: [ 'percentage' ]
+  },
```

Comment:
I _think_ that file sorts its properties alphabetically, so it might be better to move this further down.